abc.templates/infra/main.tf

```diff
@@ -4,20 +4,13 @@ locals {
 }
 
 module "github_token_minter" {
-  source = "git::https://github.com/abcxyz/github-token-minter.git//terraform?ref=f1144811e6f6583b9aa971189ca8f6ef0f947c8a"
+  source = "git::https://github.com/abcxyz/github-token-minter.git//terraform?ref=ce04ede11817b4b73c1279f9dab5ee3d49c6f7d9"
```

Comment:
I wonder if this should be a version tag also, because we will have a chicken v egg problem where we dont know the commit sha for THIS PR. So you would have to submit the changes, then submit another one just to update the sha to the previous commit. Not sure of what the best solution is. just something to be mindful of.

Reply:
I changed this to v.0.0.11 so that when we push a v0.0.11 tag it will match.